CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 ## 6.21.0 (Unreleased)
 
+NOTES:
```

Comment:
Can you carry forward the release date change note from `6.20.0`? We'll keep that until we make the change. Thanks!

Reply:
Sure. Added!